src/credentials/loginManager.ts

```diff
@@ -27,7 +27,7 @@ export class LoginManager {
      * Establishes a Credentials for the Toolkit to use. Essentially the Toolkit becomes "logged in".
      * If an error occurs while trying to set up and verify these credentials, the Toolkit is "logged out".
      */
-    public async login(credentialsProviderId: CredentialsProviderId): Promise<void> {
+    public async login(credentialsProviderId: CredentialsProviderId): Promise<string> {
```

Comment:
Why return a string? It doesn't look like the string is being processed.

Reply:
I'm uncertain for the motivation around this as well. Returning the access key id seems arbitrary, and there is another means of getting this data.

My preference is to remove this change, but if it remains, the function header should be updated.

Reply:
I ended up not using this. But why are we blocking on this? There is zero harm in returning this info.

> there is another means of getting this data.

Where? How?

Reply:
LoginManager passes the connected credentials to awsContext, where credentials can be retrieved with `getCredentials`